RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByIsolation.cc

```diff
@@ -39,6 +39,10 @@ class PFRecoTauDiscriminationByIsolation : public PFTauDiscriminationProducerBas
     calculateWeights_ = pset.exists("ApplyDiscriminationByWeightedECALIsolation") ?
       pset.getParameter<bool>("ApplyDiscriminationByWeightedECALIsolation") : false;
 
+    // RIC: allow to relax the isolation completely beyond a given tau pt
+    minPtForNoIso_ = pset.exists("minTauPtForNoIso") ?
```

Comment:
@roger-wolf  @veelken @isobelojalvo @mbluj
remind me please of the status of  ::fillDescriptions (the configuration validation system https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp )
for tau reco code.

If it's not on the tau POG list already, please add it.
Use of pset.exists is discouraged.